### PR TITLE
Trying GH actions workflows in galasa-dev org

### PR DESCRIPTION
**Pull request by jadecarino**

### CODE REVIEW

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-wrapping-pr-77-glg5t for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-wrapping-pr-77-459f5 for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-wrapping-pr-77-cgq2t for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-wrapping-pr-77-qhrlq for details. If you are unable to do so, please contact a member of the Galasa team.

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+        steps:
+          - name: Checkin Code
+            uses: actions/checkout@v4
+          
+          - name: Set up JDK 11
+            uses: actions/setup-java@v4
+            with:
+              java-version: '11'
+              distribution: 'temurin'
```
Comment (KirbyKatcher) [inline .github/workflows/build.yaml]: I think `semeru` should be used in this instance

File: .github/workflows/build.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+              server-id: 'github'
+              server-username: GITHUB_ACTOR
+              server-password: GITHUB_TOKEN
+              settings-path: '~/.m2'
+              gpg-private-key: ${{secrets.GPG_KEY}}
+              gpg-passphrase: GPG_PASSPHRASE
```
Comment (KirbyKatcher) [inline .github/workflows/build.yaml]: shouldn't this be a variable?

```yaml
  gpg-passphrase:
    description: 'Environment variable name for the GPG private key passphrase. Default is
       $GPG_PASSPHRASE.'
    required: false
```

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-wrapping-pr-77-bnjcz for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-wrapping-pr-77-l7sbn for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-wrapping-pr-77-hcvzp for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**